submitr/tests/test_rclone_support.py

```diff
@@ -0,0 +1,842 @@
+from contextlib import contextmanager
+from enum import Enum
+import json
+import os
+import pytest
+from typing import Optional, Tuple
+from unittest.mock import patch as mock_patch
+from dcicutils.file_utils import are_files_equal, compute_file_md5, normalize_path
+from dcicutils.misc_utils import create_uuid
+from dcicutils.tmpfile_utils import (
+    create_temporary_file_name, remove_temporary_file,
+    temporary_directory, temporary_file, temporary_random_file)
+from submitr.file_for_upload import FilesForUpload
+from submitr.rclone.rcloner import RCloner
+from submitr.rclone.rclone_config import RCloneConfig
+from submitr.rclone.rclone_amazon import AmazonCredentials, RCloneAmazon
+from submitr.rclone.rclone_google import GoogleCredentials, RCloneGoogle
+from submitr.rclone.rclone_utils import cloud_path
+from submitr.rclone.testing.rclone_utils_for_testing_amazon import AwsCredentials, AwsS3
+from submitr.rclone.testing.rclone_utils_for_testing_google import GcpCredentials, Gcs
+from submitr.rclone.rclone_installation import RCloneInstallation
+from submitr.s3_upload import upload_file_to_aws_s3
+from submitr.s3_utils import get_s3_key_metadata
+import submitr.submission_uploads  # noqa
+from submitr.submission_uploads import do_any_uploads  # noqa/xyzzy
+from submitr.tests.testing_rclone_helpers import (  # noqa/xyzzy
+    setup_module as rclone_setup_module, teardown_module as rclone_teardown_module,
+    load_test_data_json, Mock_LocalStorage, Mock_Portal, RANDOM_TMPFILE_SIZE)
+
+
+pytestmark = pytest.mark.integration
```

Comment:
Usually you want this to be an array in case you want to introduce additional marks later (ie: `pytest.mark.skip`)